webapp/lib/view.dart

```diff
@@ -870,6 +885,25 @@ class TagPanelView {
     }
     assert(_tagList.children.length == 0);
   }
+
+  void filterAllTags(bool showAll) {
+    for(DivElement tag in _tagList.children) {
+      if (!showAll && isAgeTag(tag)) {
+        tag.classes.toggle('action--hide', true);
+        continue;
+      }
+      tag.classes.toggle('action--hide', false);
+    }
+  }
+
+  bool isAgeTag(DivElement tag) {
```

Comment:
This feels like a bit of a workaround to a proper tagging management system - all fine, but could you add a comment describing it as such?

Reply:
Thanks @lukechurch , added a comment, please have another look